datalad/support/ansi_colors.py

```diff
@@ -39,9 +41,34 @@
 FIELD = BOLD
 
 
+def color_enabled():
+    """Check for whether color output is enabled
+
+    Color is only enabled if the terminal is interactive.
+    If the datalad.ui.color configuration setting is 'on' or 'off', then
+    respect that.
+    If the datalad.ui.color setting is 'auto' (default), then color is
+    enabled unless the environment variable NO_COLOR is defined.
+    """
+    if not ui.is_interactive:
+        return False
+
+    UIC = cfg.get_value('datalad', 'ui.color', 'auto')
```

Comment:
Thanks!
Ideally I think we should add an item (at the end, along with `datalad.ui.progressbar`) to datalad/interface/common_cfg.py and use `cfg.obtain('datalad.ui.color')` here which would then pick up default from the common cfg definition.